server/game/cards/13.5-TB/ShadowOfTheNorth.js

```diff
@@ -0,0 +1,29 @@
+const DrawCard = require('../../drawcard.js');
+
+class ShadowOfTheNorth extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Raise claim by 1',
+            max: ability.limit.perChallenge(1),
+            cardCondition: () => this.game.isDuringChallenge(),
```

Comment:
This should just be `condition`. `condition` is used at the ability level, `cardCondition` is used at the targeting level.